examples/circle_packing.ml

```diff
@@ -44,13 +45,13 @@ let overlaps (p1, r1) (p2, r2) =
 
 (* creates a random point within screen bounds *)
 let rand_point () =
-  ( Stdlib.Random.float (fst resolution *. 2.) -. fst resolution,
```

Comment:
We don't need the `Stdlib` prefix for calling the Random module. I think we can just use `Random` here.

Reply:
Just got done doing that actually!